src/content/chainlink-nodes/v1/node-config.mdx

```diff
@@ -5138,8 +5138,8 @@ If you are using FixedPriceEstimator:
 
 If you are using BlockHistoryEstimator (default for most chains):
 
-- With gas bumping disabled, it will submit all transactions with `feecap=PriceMax` and `tipcap=<calculated using past blocks>`
-- With gas bumping enabled (default for most chains) it will submit all transactions initially with `feecap = ( current block base fee * (1.125 ^ N) + tipcap )` where N is configurable by setting `EVM.GasEstimator.BlockHistory.EIP1559FeeCapBufferBlocks` but defaults to `gas bump threshold+1` and `tipcap=<calculated using past blocks>`
+- With gas bumping disabled, it will submit all transactions with `feecap=PriceMax` and `tipcap=CALCULATED_USING_PAST_BLOCKS`
+- With gas bumping enabled (default for most chains) it will submit all transactions initially with `feecap = ( current block base fee * (1.125 ^ N) + tipcap )` where N is configurable by setting `EVM.GasEstimator.BlockHistory.EIP1559FeeCapBufferBlocks` but defaults to `gas bump threshold+1` and `tipcap=CALCULATED_USING_PAST_BLOCKS`
```

Comment:
https://github.com/smartcontractkit/chainlink/blob/5bcd414622294d8142e6cfff08f8c1b0c2dc72df/core/config/docs/chains-evm.toml#L192

long line, right scroll all the way to end